shell/components/nav/Header.vue

```diff
@@ -392,7 +400,14 @@ export default {
         v-if="isSingleProduct && !isRancherInHarvester"
         :to="singleProductLogoRoute"
       >
+        <BrandImage
+          v-if="isSingleProduct.supportCustomLogo"
+          class="side-menu-logo"
+          file-name="harvester.svg"
+          :support-custom-logo="true"
+        />
```

Comment:
I think I'm missing something. Why is this being shown within element that is only displayed when `!isRancherInHarvester` is true. I wouldn't think we'd want to show a harvester badge if it's not harvester.

Reply:
Fixed